tests/e2e-inst-test.sh

```diff
@@ -149,8 +150,7 @@ for TEST in $TESTS; do
             skip_hooked_syscall=1
             continue
         fi
-        info "setting up hooked_syscall test"
-        "${TESTS_DIR}"/hooked_syscall.sh
+        ./tests/e2e-inst-signatures/scripts/hooked_syscall.sh --build --install
```

Comment:
Shouldn't we try to build in the logic determining skip_hooked_syscall? So that if we fail to build at least we skip the test? Then we only run with --install here.